tests/test_loky_backend.py

```diff
@@ -554,14 +554,13 @@ def test_compatibility_openmp(self):
                         reason="The loky process are only defined for posix.")
     @pytest.mark.parametrize("run_file", [True, False])
```

Comment:
The main parametrization is whether we run from a file or not. In the case we use `-c` to run the script, we do not set `init_main_module` to `False` as it is not required. So I think `run_file` is the right name.